lib/worker.js

```diff
@@ -121,6 +141,10 @@ Worker.prototype.run = function () {
     queue: this.queue,
     job: this.job
   }
+  if (process.env.WORKER_MONITOR) {
```

Comment:
Well at that point you don't need to check that here monitordog already does that